app/api/models/errors/CommonMtdErrors.scala

```diff
@@ -26,6 +26,7 @@ object FromDateFormatError       extends MtdError("FORMAT_FROM_DATE", "The provi
 object ToDateFormatError         extends MtdError("FORMAT_TO_DATE", "The provided toDate is invalid", BAD_REQUEST)
 object StatusFormatError         extends MtdError("FORMAT_STATUS", "The provided status is invalid", BAD_REQUEST)
```

Comment:
StatusFormatError and FormatStatusError are the same errors. One of them has to be removed, thanks.